### DISCUSSION TITLE
Does CFG_STR_CACHE_FILE_PATH need to be a full path or can it just be a file name?

### DISCUSSION THREAD

**Discussion by jsebastianms** [microsoft/cpp_client_telemetry#1246] [Q&A] [answered]
Does CFG_STR_CACHE_FILE_PATH need to be a full path or can it just be a file name? I've seen both in the repo.
I specifically want to know for iOS. I did see there is a little documentation here: https://github.com/microsoft/cpp_client_telemetry/blob/7d1a165fa4e3a63a21881f45b45bf1fcedb92c5d/docs/Offline-storage-settings.md?plain=1#L13

**Comment by lalitb** [comment, marked as answer]
@jsebastianms  Sorry missed this question.
 CFG_STR_CACHE_FILE_PATH can both be a full path or the filename. If this is filename, the directory path is as per the logic defined [here](https://github.com/microsoft/cpp_client_telemetry/blob/f1470795977588fbf5a3c25c0ab83e16c9635505/lib/utils/Utils.cpp#L100).
So for iOS, the default path is `/data/local/tmp` unless overridden by the application.
